.github/CODEOWNERS

```diff
@@ -109,11 +109,11 @@
 
 # PRLabel: %Communication - Short Codes
 # ServiceLabel: %Communication - Short Codes
-/sdk/communication/communication-short-codes/ @Arazan @whisper6284
+/sdk/communication/communication-short-codes/ @Arazan @whisper6284 @danielortega-msft
 
 # PRLabel: %Communication - Toll Free Verification
 # ServiceLabel: %Communication - Toll Free Verification
-/sdk/communication/communication-toll-free-verification/ @Arazan @whisper6284
+/sdk/communication/communication-toll-free-verification/ @whisper6284 @danielortega-msft
```

Comment:
just to confirm: removal of Arazan intentional?

Reply:
Added back. Thanks for the catch.